ipamutils/utils_test.go

```diff
@@ -24,3 +28,19 @@ func TestGranularPredefined(t *testing.T) {
 	}
 
 }
+
+func TestInitAddressPools(t *testing.T) {
+
+	// Check for Random IPAddresses in PredefinedBroadNetworks  ex: first , last and middle
```

Comment:
nit:  these aren't "Random" addresses you are checking -- they are specific boundary addresses.  The rest of your comment says so.  :)